core/src/main/resources/hudson/model/User/sidepanel.jelly

```diff
@@ -38,7 +38,7 @@ THE SOFTWARE.
       <t:actions actions="${it.propertyActions}"/>
       <t:actions actions="${it.transientActions}"/>
       <j:if test="${it.canDelete()}">
-        <l:task confirmationMessage="${%delete.user(it.displayName)}" href="${rootURL}/${it.url}/doDelete" icon="icon-edit-delete icon-md" title="${%Delete}" post="true" requiresConfirmation="true"/>
+        <l:delete urlPrefix="${rootURL}/${it.url}" title="${%Delete}" message="${%delete.user(it.displayName)}"/>
```

Comment:
unsure where the change has happened but commenting here as related, if I click on the delete button from this page http://localhost:8080/jenkins/securityRealm/

I get redirected to:
http://localhost:8087/jenkins/securityRealm/user/john/delete

which is a 404 now:
<img width="1067" alt="image" src="https://github.com/jenkinsci/jenkins/assets/21194782/cf7913d8-0925-4e42-997d-24b4901b1e73">

Reply:
That was a bug that was fixed with https://github.com/jenkinsci/jenkins/pull/8131